data/src/main/java/io/gnosis/kouban/data/repositories/SafeRepository.kt

```diff
@@ -74,6 +74,30 @@ class SafeRepository(
             }
         }
 
+    suspend fun checkSafe(address: Solidity.Address): Boolean {
+        val responses = jsonRpcApi.post(listOf(buildMasterCopyRequest(address, 0)))
```

Comment:
There is also a version for a single request (so a list should not be required) ... but makes no big difference